### PR TITLE
.github: update CODEOWNERS

### PR DESCRIPTION
**Pull request by knz**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -17,7 +17,8 @@
 /pkg/migration/sqlmigrations @cockroachdb/core @cockroachdb/sql-wiring
 
 /pkg/ccl/                    @cockroachdb/sql-ccl
-/pkg/cli/                    @cockroachdb/sql-ui
+/pkg/cli/                    @cockroachdb/cli
+/pkg/cli/sql*                @cockroachdb/sql-ui @cockroachdb/cli
```
Comment (tamird) [inline .github/CODEOWNERS]: didn't we decide not to use asterisks? @benesch

Comment (knz) [inline .github/CODEOWNERS]: These are not directories but two files (sql.go / sql_util.go)

Comment (tamird) [inline .github/CODEOWNERS]: OK, but if someone adds a directory, do you want them to have to update this again?

Comment (knz) [inline .github/CODEOWNERS]: aha I see what you mean. I extended the glob to make the intent clearer.

**Review by tamird — APPROVED** [review]

**Pull request merged**